coins/coins.go

```diff
@@ -105,6 +105,20 @@ func (a *PiconeroAmount) AsMonero() *apd.Decimal {
 	return xmrAmt
 }
 
+// AsMoneroString converts a PiconeroAmount into a formatted XMR amount string.
+func (a *PiconeroAmount) AsMoneroString() string {
+	xmrAmt := new(apd.Decimal).Set(a.Decimal())
+	decreaseExponent(xmrAmt, NumMoneroDecimals)
+	_, _ = xmrAmt.Reduce(xmrAmt)
+	return xmrAmt.Text('f')
```

Comment:
TODO: This function should just be `return a.AsMonero().Text('f')` and the function below should call this one.  I'll fix it before merging.